src/rgw/rgw_file_int.h

```diff
@@ -226,6 +226,7 @@ namespace rgw {
       ~file();
     };
 
+    // coverity[missing_lock:SUPPRESS]
```

Comment:
why is the lock supression before the definition of the struct?
which data member is not protected?